package.json

```diff
@@ -77,7 +77,7 @@
     "rimraf": "^2.6.3",
     "rollup": "^2.0.0",
     "sinon": "^7.2.2",
-    "typescript": "^3.8.3",
+    "typescript": "3.9.7",
```

Comment:
We now fix the version?

Reply:
Yeah I think that would be a smart thing, since any Typescript minor or even patch can be breaking, due to the nature of Typescript where every change is essentially always breaking